_ont/ont-huawei-ma5671a-root.md

```diff
@@ -111,21 +111,23 @@ sed -i  "s|/opt/lantiq/bin/minishell|/bin/ash|g" /etc/passwd
 {% include alert.html content="The cause of these kernel panics could be insufficient power supply." alert="Info"  icon="svg-info" color="blue" %}
 
 {:style="counter-reset:none"}
-8. Reboot it this time connected to the router with cage or mediaconverter, with the port set to an IP on the 192.168.1.0/24 subnet (the stick has the IP 192.168.1.10)
+9. Reboot it this time connected to the router with cage or mediaconverter, with the port set to an IP on the 192.168.1.0/24 subnet (the stick has the IP 192.168.1.10)
 
 {% include alert.html content="If your subnet is 192.168.1.0/24 make sure you have no ip conflicts." alert="Note"  icon="svg-warning" color="yellow" %}
 
 {:style="counter-reset:none"}
-9. Run the terminal and login to the stick with ssh
+10. Run the terminal and login to the stick with ssh
 
 ```shell
 ssh root@192.168.1.10
 ```
 
 The password is `admin123`.
 
+{% include alert.html content="If you use OpenSSH >= 8.8 you will have to enable some deprecated algorithms: ssh -oKexAlgorithms=+diffie-hellman-group1-sha1 -oHostKeyAlgorithms=+ssh-dss [...]" alert="Info"  icon="svg-info" color="blue" %}
```

Comment:
I'm not sure this is needed since 8.8, I think it was introduced earlier.
While the -O option is required starting from 8.8

Reply:
How about "If you use a modern OpenSSH version (e.g. >= 8.8) [...]"